src/OtherQuantityHandler.cpp

```diff
@@ -199,6 +199,13 @@ void OtherQuantityHandler::DefineQuantitiesSTREAM() {
         for (len_t ir = 0; ir < nr; ir++)
             v[ir] = 1/this->confinementTime->EvaluatePerpendicularConfinementTime(ir);
     );
+    
+    // MODIFICATION
```

Comment:
comments like this are not necessary when version controlling codes...

Reply:
Done